polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaSource.scala

```diff
@@ -287,10 +287,11 @@ class ScalaSource[Interpreter <: ScalaInterpreter](val interpreter: Interpreter)
         }
 
         for {
-          tpe     <- withCompiler(qual.tpe.widen)
-          scope    = addContext(tpe.members)
+          tpe     <- withCompiler(qual.tpe)
+          widened <- if (tpe != null) withCompiler(tpe.widen) else Right(global.NoType)
```

Comment:
Is this change mainly so we can check for null here or is there something else it's doing?

Reply:
Just so it can check for `qual.tpe` being null